configs/projects/agent-runtime.rb

```diff
@@ -0,0 +1,69 @@
+project 'agent-runtime' do |proj|
+  platform = proj.get_platform
+  # TODO: Is this an acceptable way to inherit from multiple possible agent version branches?
+  proj.setting(:agent_branch, ENV['AGENT_BRANCH'] || 'master')
+
+  proj.inherit_settings 'puppet-agent', 'git://github.com/puppetlabs/puppet-agent', proj.settings[:agent_branch]
+  # Set the version based on `git describe`, then prepend the branch name
+  # TODO: (It seems like there must be a better way to name the package - Is this valuable?)
```

Comment:
I don't think you want to be using `package_version` here, that will be the version from agent.

I think if we go forward with a single agent project for all the branches you probably want to replace the `proj.version` line below with
`proj.version "#{proj.settings[:agent_branch]}-#{proj.get_version}"`